test/integration/user_login_test.rb

```diff
@@ -21,4 +25,14 @@ class UserLoginTest < ActionDispatch::IntegrationTest
       assert page.has_content?('Welcome, user')
     end
   end
+
+  test 'a user can logout' do
+    fill_in 'session[username]', with: 'user'
+    fill_in 'session[password]', with: 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.